spec/lib/bamboozled/api/employee_spec.rb

```diff
@@ -137,4 +137,56 @@
       expect(employee["headers"]).to eq(expected_headers)
     end
   end
+
+  context 'when an API BASE is set' do
+    before do
+      @client = Bamboozled.client(
+        subdomain: "subdomain", api_key: "new_api_key", api_url: 'https://new.bamboo/url/'
+      )
+    end
+
+    it 'gets all employees from the new API' do
+      response = File.new("spec/fixtures/all_employees.json")
+      stub_request(:get, "https://new_api_key:x@new.bamboo/url/employees/directory").to_return(response)
+      employees = @client.employee.all
+
+      expect(employees).to be_a Array
+      expect(employees.first.count).to eq 7
+      expect(1).to eq 1
+    end
+
+    describe '#update' do
```

Comment:
Same comment about single/double quotes

Reply:
I copied the structure of the tests the way it is and adapted it. That's why is like this